GameServer/spells/AssassinateSpellHandler.cs

```diff
@@ -0,0 +1,72 @@
+using DOL.Database;
+using DOL.Events;
+using DOL.GS.PacketHandler;
+using DOL.GS.Styles;
+using DOL.Language;
+using System;
+
+namespace DOL.GS.Spells
+{
+    [SpellHandler("Assassinate")]
+    public class AssassinateHandler : SpellHandler
```

Comment:
"Could not Instanciate Ability Assassinate from DOL.GS.RealmAbilities.AssasinateAbility".

Reply:
As you can see there is a mistake in the table "Abilities" regarding "AssasinateAbility". I've fixed this mistake in the last release of DOL PUBLIC DB here : https://github.com/Eve-of-Darkness/db-public/pull/222/commits/75abb481980ae7884b9b341f7e17528eb16f8737

Reply:
As you can see there is a mistake in the table "Abilities" regarding "AssasinateAbility". I've fixed this mistake in the last release of DOL PUBLIC DB here : Eve-of-Darkness/db-public@75abb48

Reply:
Thank you for the reply. Missed the typo :sweat_smile:

Reply:
I was actually under the impression this is now a regular ability in Stealth skill, but it was actually removed from Live. This makes it harder to reason about this. The ability itself should be casted while being stealthed I think (currently breaks stealth).